packages/components/CHANGELOG.md

```diff
@@ -16,7 +16,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 #### Changed
 
-- `Spinner`, `Link Pure`, `Button Pure`, `Checkbox Wrapper`, `Radio Button Wrapper`, `Tag Dismissible` matches new design language
+- `Spinner`, `Link Pure`, `Button Pure`, `Checkbox Wrapper`, `Radio Button Wrapper`, `Tag Dismissible`, `Tag` matches
+  new design language
+- Color `notification-neutral` is renamed to `notification-information` for `Tag`
```

Comment:
Please add a diff for breaking changes so people know what to do.

Reply:
Added diff